src/app/src/components/DashboardMergeFacilities.jsx

```diff
@@ -0,0 +1,181 @@
+    fetchToMergeFacilityOnEnterKeyPress,
+    resetMergeState,
+    mergeSelectedFacilities,
+    merging,
+    errorMerging,
+    flipSelectedFacilities,
+}) {
+    /* eslint-disable react-hooks/exhaustive-deps */
+    useEffect(() => resetMergeState, []);
+    /* eslint-enable react-hooks/exhaustive-deps */
+
+    return (
+        <>
+            <DashboardMergeFacilityControls
+                handleMerge={mergeSelectedFacilities}
+                handleFlip={flipSelectedFacilities}
+                merging={merging}
+                error={errorMerging}
+                disabled={
+                    targetFetching || toMergeFetching || !targetData || !toMergeData
+                }
+                toMergeData={toMergeData}
+                targetData={targetData}
+            />
+            <div style={{ width: '100%', display: 'flex' }}>
+                <DashboardFacilityCard
+                    updateOARID={updateTargetOARID}
+                    fetchFacility={fetchTargetFacility}
+                    clearFacility={clearTargetFacility}
+                    oarID={targetOARID}
+                    data={targetData}
+                    fetching={targetFetching}
+                    error={targetError}
+                    handleEnterKeyPress={fetchTargetFacilityOnEnterKeyPress}
+                    title="Target facility for merge"
+                />
+                <DashboardFacilityCard
+                    updateOARID={updateToMergeOARID}
+                    fetchFacility={fetchToMergeFacility}
+                    clearFacility={clearToMergeFacility}
+                    oarID={toMergeOARID}
+                    data={toMergeData}
+                    fetching={toMergeFetching}
+                    error={toMergeError}
+                    handleEnterKeyPress={fetchToMergeFacilityOnEnterKeyPress}
+                    title="Facility to merge into target"
+                />
+            </div>
+        </>
+    );
+}
+
+DashboardMergeFacilities.defaultProps = {
+    targetData: null,
+    targetError: null,
+    toMergeData: null,
+    toMergeError: null,
+    errorMerging: null,
+};
+
+DashboardMergeFacilities.propTypes = {
+    targetOARID: string.isRequired,
+    targetData: facilityDetailsPropType,
+    targetFetching: bool.isRequired,
+    targetError: arrayOf(string),
+    toMergeOARID: string.isRequired,
+    toMergeData: facilityDetailsPropType,
+    toMergeFetching: bool.isRequired,
+    toMergeError: arrayOf(string),
+    updateTargetOARID: func.isRequired,
+    clearTargetFacility: func.isRequired,
+    fetchTargetFacility: func.isRequired,
+    fetchTargetFacilityOnEnterKeyPress: func.isRequired,
+    updateToMergeOARID: func.isRequired,
+    clearToMergeFacility: func.isRequired,
+    fetchToMergeFacility: func.isRequired,
+    fetchToMergeFacilityOnEnterKeyPress: func.isRequired,
+    resetMergeState: func.isRequired,
+    mergeSelectedFacilities: func.isRequired,
+    merging: bool.isRequired,
```

Comment:
There is a prop type warning here when first loading the merge page

<img width="1153" alt="Screen Shot 2019-06-19 at 2 31 56 PM" src="https://user-images.githubusercontent.com/17363/59803004-1584f080-929f-11e9-9118-41b482339ff4.png">

Reply:
I mistakenly did not rename `fetching` to `merging` in the destructuring block originally. 388f9afc3ad4033f56158e4db1ea070267bccbac fixes that.